packages/shared/src/constants/common.ts

```diff
@@ -12,4 +12,4 @@ export const DEFAULT_NS = 'default';
 export const RACK_LABEL = 'topology.rook.io/rack';
 export const NOOBA_EXTERNAL_PG_SECRET_NAME = 'noobaa-external-pg';
 export const NOOBAA_EXTERNAL_PG_TLS_SECRET_NAME = 'noobaa-external-pg-tls';
-export const PLUGIN_VERSION = process.env.PLUGIN_VERSION;
+export const PLUGIN_VERSION = process?.env?.PLUGIN_VERSION;
```

Comment:
E2E tests were always failing, so I saw the videos and reproduced locally:
![Screenshot from 2024-07-30 08-58-10](https://github.com/user-attachments/assets/c20e2a4a-d6be-4ba8-8066-33844ccaa160)
From the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining):
![image](https://github.com/user-attachments/assets/ff41e614-de8c-4c98-99f8-6eb1335cb81b)

I tested this solution locally and it works:
```suggestion
export const PLUGIN_VERSION = typeof process === 'undefined' ? undefined : process?.env?.PLUGIN_VERSION;
```

Reply:
@SanjalKatiyar ping.